### PR TITLE
Router

### PR DESCRIPTION
**Pull request by ThomasDruart**
Router  updated with Hash-Link on "about-us" page for smooth transitions.
Npx prettier done

### CODE REVIEW

File: src/components/Atelier/index.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { Link } from "react-router-dom";
 export default function Atelier() {
   return (
     <Container>
-      <Image src="/images/atelierbis.jpg" alt="atelier" />
+      <Image src={"/images/Atelier.jpg"} alt={"atelier"} />
```
Comment (AmjadLille) [inline src/components/Atelier/index.jsx]: {} ??

File: src/components/Header/Navbar/index.jsx [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
+            </HashLink>
+            <HashLink smooth to="/about#partenaires">
+              Nos Partenaires
+            </HashLink>
           </DropdownItems>
         </NavDropdown>
         |
         <NavDropdown>
-          <a href="/action/pedagogie">Nos Actions</a>
+          <a href="#!">Nos Actions</a>
```
Comment (AmjadLille) [inline src/components/Header/Navbar/index.jsx]: - replace a tag by Link.
- Loic will not accept "#!"
- last commit Prettify

**Review by AmjadLille — CHANGES_REQUESTED** [review]

**Comment by ThomasDruart** [comment]
Changes requested done.

**Review by AmjadLille — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**